src/main/java/be/seeseemelk/mockbukkit/entity/EntityMock.java

```diff
@@ -372,13 +371,13 @@ public boolean teleport(@NotNull Entity destination, @NotNull TeleportCause caus
 	@Override
 	public boolean isOp()
 	{
-		return operator;
+		return false;
 	}
 
 	@Override
 	public void setOp(boolean isOperator)
 	{
-		operator = isOperator;
+		//CraftBukkit does nothing here
```

Comment:
This should throw an UnsupportedOperationException. There's no reason anyone should be setting an entity to op, and if they are, the test should fail since it doesn't do anything, and them thinking it does could lead to unexpected behavior.